tools/shared.py

```diff
@@ -412,31 +412,29 @@ def fix_js_engine(old, new):
   return new
 
 
-def expected_llvm_version():
+def supported_llvm_versions():
```

Comment:
perhaps a comment that they must be in order (as the callers seem to depend on that)?